packages/mjml-button/src/index.js

```diff
@@ -51,85 +92,116 @@ class Button extends Component {
   styles = this.getStyles()
 
   getStyles () {
-    const { mjAttribute, defaultUnit } = this.props
+    const {mjAttribute} = this.props,
+      width = mjAttribute('width'),
```

Comment:
Oh ok, is this considered a bad practice?

`Should we add a new attribute to allow current button behaviour ?`
Hmm, I don't think anyone would like the previous behaviour over the "normal" one. At worst, they won't care IMO.